perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/columnProviders/OverviewAttributesColumnProvider.java

```diff
@@ -0,0 +1,199 @@
+package cz.metacentrum.perun.wui.model.columnProviders;
+
+import com.google.gwt.cell.client.FieldUpdater;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.core.client.JsArrayString;
+import com.google.gwt.json.client.JSONValue;
+import cz.metacentrum.perun.wui.client.resources.PerunTranslation;
+import cz.metacentrum.perun.wui.model.ColumnProvider;
+import cz.metacentrum.perun.wui.model.GeneralObject;
+import cz.metacentrum.perun.wui.model.beans.Attribute;
+import cz.metacentrum.perun.wui.model.resources.PerunComparator;
+import cz.metacentrum.perun.wui.widgets.PerunDataGrid;
+import cz.metacentrum.perun.wui.widgets.resources.PerunColumn;
+import cz.metacentrum.perun.wui.widgets.resources.PerunColumnType;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Implementation of {@link ColumnProvider ColumnProvider}
+ * for {@link Attribute Attribute} to display user information.
+ *
+ * @author Vojtěch Sassmann <vojtech.sassmann@gmail.com>
+ */
+public class OverviewAttributesColumnProvider extends ColumnProvider<Attribute> {
```

Comment:
How and why does it differ from existing ``AttributesColumnProvider`` ? Please merge this logic into single class or express necessary differencies. If logic is not shared for all Attribute objects, you should move it from core to profile.

Reply:
I have merged the two classes into one. The difference was that the old one was showing names as links. I have added mechanism so you can choose if you want this behavior or not.